include/rosdiscover-clang/Stmt/ApiCall.h

```diff
@@ -78,12 +78,14 @@ class Subscriber : public SymbolicRosApiCall {
     return {
       {"kind", "subscribes-to"},
       {"name", getName()->toJson()},
-      {"format", format}
+      {"format", format},
+      {"callback", callback->toJson()}
```

Comment:
callback can be null (see line 28 of [SubscribeTopicCall.h](https://github.com/rosqual/rosdiscover-cxx-recover/pull/23/files#diff-d22fec85d4287ccc92525d6e456fb0cbc3fbb0c45eb8fc137144562591c1ee85)), so this will lead to a null dereference. You can get around this by checking that callback isn't nullptr before adding it to the JSON description